browser/src/VariantPage/LiftoverDisambiguationPage.tsx

```diff
@@ -0,0 +1,140 @@
+
+  return (
+    <TrackPage>
+      <TrackPageSection>
+        <DocumentTitle
+          title={`${fromVariantId} liftover | ${fromReferenceGenome} to ${toReferenceGenome}`}
+        />
+        <>
+          Due to liftover, variant {fromVariantId} in dataset {fromLabel} may correspond to a
+          different variant or variants in {toReferenceGenome}.
+        </>
+        <BaseQuery
+          operationName="LiftoverDisambiguation"
+          query={disambiguationQuery}
+          variables={queryVariables}
+        >
+          {({ data, error, graphQLErrors, loading }: any) => {
+            let pageContent = null
+            if (loading) {
+              pageContent = (
+                <Delayed>
+                  <StatusMessage>Loading corresponding variants...</StatusMessage>
+                </Delayed>
+              )
+            } else if (error) {
+              pageContent = <StatusMessage>Unable to load corresponding variants</StatusMessage>
+            } else if (!(data || {}).liftover) {
+              pageContent = (
+                <StatusMessage>
+                  {graphQLErrors && graphQLErrors.length
+                    ? Array.from(
+                        new Set(
+                          graphQLErrors
+                            .filter((e: any) => !e.message.includes('ClinVar'))
+                            .map((e: any) => e.message)
+                        )
+                      ).join(', ')
+                    : 'Unable to load corresponding variants'}
+                </StatusMessage>
+              )
+            } else if (data.liftover.length === 0) {
+              pageContent = <StatusMessage>No corresponding variants found</StatusMessage>
+            } else if (data.liftover.length === 1) {
+              pageContent = (
+                <Redirect
+                  to={{
+                    pathname: `/variant/${data.liftover[0][correspondingVariantField].variant_id}`,
+                    search: `dataset=${baseDatasetForReferenceGenome(toReferenceGenome)}`,
+                  }}
+                />
+              )
+            } else {
+              pageContent = (
+                <>
+                  <ul>
+                    {data.liftover.map(
+                      (
+                        correspondingVariant: Record<
+                          CorrespondingVariantField,
+                          { variant_id: string } | undefined
+                        >
+                      ) => {
+                        const variantId =
+                          correspondingVariant[correspondingVariantField]!.variant_id
+                        const datasetToLink = baseDatasetForReferenceGenome(toReferenceGenome)
+                        return (
+                          <li key={variantId}>
+                            <Link to={`/variant/${variantId}?dataset=${datasetToLink}`}>
+                              {variantId}
+                            </Link>
```

Comment:
When I was manually checking here (set the conditionals above to always be false so this branch triggers) this link is broken for me, it appends the new dataset on top of the v2 dataset from where it came from (i.e. directed to "http://localhost:8008/variant/1-55039779-T-G?dataset=gnomad_r4?dataset=gnomad_r2_1"), this double query param results in an error page.

I believe the `Link` component has a default value for a prop called `preserveSelectedDataset` as true, to keep the dataset query param when navigating between pages. 

https://github.com/broadinstitute/gnomad-browser/blob/d9085aa14f846705598d80cabfcae813ee7e0cbb/browser/src/Link.tsx#L22-L45

As such, just passing false to this will fix the link.

```
<Link
  to={`/variant/${variantId}?dataset=${datasetToLink}`}
  preserveSelectedDataset={false}
>
  {variantId}
</Link>